build/ci/.azure-devops-package-generic.yml

```diff
@@ -83,6 +83,14 @@ jobs:
       logProjectEvents: false
       createLogFile: false
 
+  - task: CmdLine@2
+    inputs:
+      workingDirectory: $(build.artifactstagingdirectory)
+      script: |
+        dotnet tool install --global Basic.CompilerLog
+        complog create --projectName=Uno.UI build-$(GitVersion.FullSemVer)-$(XAML_FLAVOR_BUILD).binlog
+    condition: always()
+
```

Comment:
@jeromelaban Let me know if you want me to keep this, delete it, or comment it.

Reply:
Let's comment it out for now, but it will be very useful for some advanced debugging,